src/base/SearchBar.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const StyledInputContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+`;
+
+const StyledInputBox = styled.input`
+  width: 85%;
+  margin: 10px;
+  border-radius: 5px;
+  border: none;
+  background-color: rgba(0, 0, 0, 0);
+  font-size: 18px;
+  padding: 5px 10px;
+  transition: all 0.25s ease-in;
+  font-weight: lighter;
+
+  &:focus {
+    outline: none;
+  }
+
+  &::placeholder {
+    font-weight: 100;
+  }
+`;
+
+const InputBoxClear = styled.div`
+  font-size: 11px;
+  margin: auto;
+  transition: font-size 0.1s ease-in-out;
+  color: gray;
+
+  &:hover {
+    cursor: pointer;
+    font-size: 15px;
+  }
+`;
+
+const SearchBar = (props) => {
+  return (
+    <StyledInputContainer>
+      <StyledInputBox
+        placeholder="검색하기"
+        onChange={props.onInputChange}
+        value={props.value}
+      />
+      <InputBoxClear onClick={props.onInputReset}>✕</InputBoxClear>
+    </StyledInputContainer>
+  );
+};
```

Comment:
컴포넌트가 어떤 props를 가지는지에 대해서 정리해주면 더 좋을 것 같아요.
맨 첫줄을 

```jsx
const SearchBar = ({value, onInputReset, onInputChange}) => {
```

혹은

```suggestion
const SearchBar = (props) => {
  const { onInputchange, onInputReset, value } = props; // 이렇게 따로 명시
  return (
    <StyledInputContainer>
      <StyledInputBox
        placeholder="검색하기"
        onChange={props.onInputChange}
        value={props.value}
      />
      <InputBoxClear onClick={props.onInputReset}>✕</InputBoxClear>
    </StyledInputContainer>
  );
};
```

이렇게 해놓고 컴포넌트 객체의 위치를 파일 상단으로 올리면 파일을 열자마자 컴포넌트의 기능은 뭔지, 어떤 props를 받는지 한번에 볼 수 있겠죠?